src/main/java/seedu/siasa/logic/commands/contact/SortContactCommand.java

```diff
@@ -40,7 +42,11 @@ public CommandResult execute(Model model) {
         requireNonNull(model);
         if (comparator != null) {
             model.updateFilteredContactList(comparator);
-            return new CommandResult(MESSAGE_SUCCESS);
+            if (model.getFilteredPolicyList().isEmpty()) {
+                return new CommandResult(MESSAGE_NO_POLICIES);
+            } else {
+                return new CommandResult(MESSAGE_SUCCESS);
+            }
```

Comment:
Is there a better way to do this, especially since it is repeated a few times?

If not can consider making the Happy Path more prominent.

Reply:
Resolved at 26a10b30c1fd5c1733a107a45c42a06817cb2a85